pyproject.toml

```diff
@@ -31,18 +31,17 @@ classifiers = [
     "Operating System :: Microsoft :: Windows",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Topic :: Scientific/Engineering",
     "Topic :: Scientific/Engineering :: Bio-Informatics",
     "Topic :: Scientific/Engineering :: Chemistry",
     "Topic :: Software Development :: Libraries :: Python Modules"
 ]
-requires-python = ">=3.10"
+requires-python = ">=3.11"
 
 dependencies = [
-	     "numpy",
+	     "numpy<2.3",
```

Comment:
If we only support numpy up to a specific version then we need to state it together with a reason in the CHANGELOG.

Reply:
The numpy issue shows as an example corner cases with nan. We are seeing issues in normal calculations. 

Do we know if this is ultimately pymbar or alchemlyb?

Could you please post (in the main comments) the output when our tests run with numpy 2.3 so that I can get a sense of differences? At the face of it, I don't want my free energies to differ by 10% (rtol=0.1) just because I switched to a new version.

Reply:
100% pymbar madness https://github.com/choderalab/pymbar/issues/556

Reply:
Thanks for checking and raising the issue in pymbar!

Reply:
Yikes for the differences. 😱